src/components/atoms/charts/Calendar3D/index.jsx

```diff
@@ -0,0 +1,375 @@
+      datesCurrent = "-";
+    }
+
+    return {
+      __html: `<div class="ic-stats-block ic-stats-bottom">\n
+    <span class="ic-stats-table">\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">Longest streak\n
+    <span class="ic-stats-count">${streakLongest}</span>\n
+    </span>\n
+    <span class="ic-stats-meta">\n
+    <span class="ic-stats-unit">days</span>\n
+    <span class="ic-stats-date">${datesLongest}</span>\n
+    </span>\n
+    </span>\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">Current streak\n
+    <span class="ic-stats-count">${streakCurrent}</span>\n
+    </span>\n
+    <span class="ic-stats-meta">\n
+    <span class="ic-stats-unit">days</span>\n
+    <span class="ic-stats-date">${datesCurrent}</span>\n
+    </span>\n
+    </span>\n
+    </span>\n
+    </div>`,
+    };
+  }
+
+  renderIsometrics = () => {
+    if (this.context) {
+      // Create a canvas 2D point for pixel view world
+      let point = new obelisk.Point(70, 70);
+      // Create view instance to nest everything
+      // Canvas could be either DOM or jQuery element
+      let pixelView = new obelisk.PixelView(this.context, point);
+
+      pixelView.clear();
+
+      // Get contributions of the selected year
+      let contribData;
+
+      if (this.props.year) {
+        contribData = this.props.platformData.statistic.years.find(
+          (element) => element.year === this.props.year
+        );
+      } else {
+        contribData = this.props.platformData.statistic.current;
+      }
+
+      let contributions = contribData.calendar;
+
+      // Define basic variables
+      let size = 2 * Math.round(this.state.width / 80 / 2);
+
+      if (size <= 8) {
+        size = 8;
+      }
+
+      const maxHight = 100;
+
+      let x = 0;
+      let y = 0;
+      let maxCount = 0; // Max number of contributions / day in the last year
+      let values = [];
+
+      contributions.weeks.map((week, wkey) => {
+        values[wkey] = [];
+
+        week.days.map((day, dkey) => {
+          // Get max number of contributions
+          if (day.total > maxCount) {
+            maxCount = day.total;
+          }
+          values[wkey][dkey] = day;
+        });
+      });
+
+      values.map((week, wi) => {
+        week.map((day, di) => {
+          // Normalize the values to achieve even distribution
+          let cubeHeight = 3;
+
+          if (maxCount > 0) {
+            cubeHeight += parseInt((maxHight / maxCount) * day.total);
+          }
+
+          // Offsets
+          let x = wi;
+          let y = di;
+
+          // Create cube dimension and color instance
+          let fill = day.color;
+          let color = new obelisk.CubeColor().getByHorizontalColor(
+            parseInt("0x" + fill.replace("#", ""))
+          );
+
+          // ANIMATION TOGGLE for kleberbaum to play with
+          const animated = false;
+
+          if (animated) {
+            var animHeight = 3;
```

Comment:
WTF, @Aichnerc isn't using var in JS also "considered a war crime"?
LMAO xD
```suggestion
            let animHeight = 3;
```

Reply:
Have a look:
https://github.com/snek-at/intel/pull/78#discussion_r436346039

Reply:
Indeed.